subcommittees/mentoring/roles/mentors.md

```diff
@@ -0,0 +1,15 @@
+# Mentors
+
+## Responsibilities
+
```

Comment:
- Decide on a meeting schedule with your Mentees.  
- Personalize meetings based on Mentee’s needs and desires.  
- Respond promptly to emails from your Mentees.  
- Follow up with Mentees who miss meetings.  
- Be respectful of Mentees. Provide a safe space for questions and mistakes.  
- Report Mentee attendance and progress to mentoring coordinator.